src/components/App.tsx

```diff
@@ -92,9 +97,22 @@ export const App: FunctionalComponent<{ label: string }> = (attributes, nodes, u
 						.filter(child => child.vattrs?.path != undefined)
 						.map(child =>
 							child.vattrs?.to ? (
-								<Route path={resolve(child.vattrs?.path) ?? child.vattrs?.path} to={child.vattrs?.to}></Route>
+								<Route
+									path={
+										typeof child.vattrs?.path == "string"
+											? resolve(child.vattrs?.path)
+											: child.vattrs?.path ?? child.vattrs?.path
```

Comment:
`: child.vattrs?.path ?? child.vattrs?.path` isnt this repeated?
why not `typeof child.vattrs?.path == "string" ? resolve(child.vattrs?.path) : child.vattrs?.path`